src/libraries/System.Security.Cryptography/src/System/Security/Cryptography/X509Certificates/OpenSslCachedSystemStoreProvider.cs

```diff
@@ -361,24 +366,24 @@ private static string[] GetRootStoreDirectories(out bool isDefault)
             return directories;
         }
 
-        private static bool TryStatFile(string path, out DateTime lastModified)
-            => TryStat(path, Interop.Sys.FileTypes.S_IFREG, out lastModified);
-
         private static bool TryStatDirectory(string path, out DateTime lastModified)
-            => TryStat(path, Interop.Sys.FileTypes.S_IFDIR, out lastModified);
+            => TryStat(path, Interop.Sys.FileTypes.S_IFDIR, out lastModified, out _);
 
-        private static bool TryStat(string path, int fileType, out DateTime lastModified)
+        private static bool TryStatFile(string path, out DateTime lastModified, out (long, long) fileId)
+            => TryStat(path, Interop.Sys.FileTypes.S_IFREG, out lastModified, out fileId);
+
+        private static bool TryStat(string path, int fileType, out DateTime lastModified, out (long, long) fileId)
         {
```

Comment:
Ideally the TryStatFile/TryStatDirectory order would have been maintained so the diff showed that it was really just adding the out.  But, again, it's not worth spinning on.